packages/common/src/ui/components/domain/StatusCrumbs/StatusCrumbs.tsx

```diff
@@ -2,97 +2,72 @@ import React from 'react';
 import Breadcrumbs from '@mui/material/Breadcrumbs';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
-
-import { useAppTheme } from '../../styles';
-import { ChevronDownIcon } from '../icons';
+import { ChevronDownIcon } from '../../../icons';
```

Comment:
Moved everything into a folder in `/components` because it seems neat. Then I had trouble naming things, of course.